docs/frontend/ajax-client.md

```diff
@@ -0,0 +1,127 @@
+# AJAX client
+
+The AJAX client is a simple wrapper around [`Axios`](https://axios-http.com/).
+
+## Default Axios Config
+
+Some configuration is done by default. But it can be overruled.
+
+* `timeout`: 2500
+* `headers.common`: `Accept: application/json`
+
+## Usage
+
+Is the AJAX Client is just an extended version of the Axios client all Axios
```

Comment:
Hier staat een 'is' te veel :)

Reply:
Fixed in cc3b69746a91e28198c4e00c2b74ca35f986bb22